cylc/flow/network/ssh_client.py

```diff
@@ -69,6 +93,28 @@ def send_request(self, command, args=None, timeout=None):
         Returns:
             object: Deserialized output from function called.
         """
+        cmd, ssh_cmd, login_shell, cylc_path, message = self.prepare_command(
+            command, args, timeout)
+        proc = _remote_cylc_cmd(
+            cmd,
+            host=self.host,
+            stdin_str=message,
+            ssh_cmd=ssh_cmd,
+            remote_cylc_path=cylc_path,
+            ssh_login_shell=login_shell,
+            capture_process=True)
```

Comment:
In the ZMQ client we avoid duplication in a really lazy way by always calling the async interface:

```python
task = self.loop.create_task(                                                      
    self.async_request(command, args, timeout))          
self.loop.run_until_complete(task)          
return task.result()   
```

Might be a good idea to prevent these interfaces from diverging.

Reply:
Ok, I've had a go at implementing this. I've had to start new `asyncio.loop` as the standard client.py I think is using the loop from ZMQ which we don't have here. See what you think, shout if there is a better way that I haven't spotted!